src/components/lib/posix/posix.c

```diff
@@ -513,10 +533,26 @@ cos_syscall_handler(int syscall_num, long a, long b, long c, long d, long e, lon
 		return cos_syscalls[syscall_num](a, b, c, d, e, f);
 	}
 }
+/* TODO: init tls when creating components */
+char tls_space[8192] = {0};
+void tls_init()
```

Comment:
This is also an empty API. I might miss something. How do you get tls working?

Reply:
Please ignore this part because I cannot remove this within this PR. This is still under tests.